smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/SwiftWriter.kt

```diff
@@ -111,8 +120,8 @@ class SwiftWriter(private val fullPackageName: String, swiftImportContainer: Swi
 
     // Adds an import statement that imports the individual type from the specified module
     // Example: addIndividualTypeImport("struct", "Foundation", "Date") -> "import struct Foundation.Date"
-    fun addIndividualTypeImport(kind: String, module: String, type: String) {
-        importContainer.addImport("$kind $module.$type", false)
+    fun addIndividualTypeImport(kind: SwiftKind, module: String, type: String) {
+        importContainer.addImport("${kind.kind} $module.$type", false)
     }
```

Comment:
Modify function to use `SwiftKind` that codifies the _kind_ (e.g., typealias, class, let, etc.) being imported.